kit/blueprint/styles.scss

```diff
@@ -67,6 +67,7 @@ $pt-transition-duration: 100ms !default;
 // This override is taken from the popover source SCSS - there it is scoped under the minimal class.
 // https://github.com/palantir/blueprint/blob/1bfd1e42303d2626bfc3923eec2195ab4dc696d2/packages/core/src/components/popover/_popover.scss#L54
 .bp5-popover {
+  margin: 0px 5px;
```

Comment:
Comment above this selector is now misleading, as this new part doesn't relate to the override taken from source.

This is avoiding an issue where popper will place a menu up against the very edge of the viewport - i.e. its overflow constraints will adjust the positioning to ensure it doesn't go past the boundary, but won't add any more additional padding by default. 

My concern is that this is now applied to all popovers, even those not near the edge, and might be a surprising override or cause some unexpected issue with someone looking to place a popover precisely.  

From the popper docs, it looks like the correct solution might be to use the `padding` option on the `preventOverflow` modifier - https://popper.js.org/docs/v2/modifiers/prevent-overflow/

BP installs that modifier by default, but lets you customize - https://github.com/palantir/blueprint/blob/0f8d6d9689e42ca09d36b3f63c564e19f485b91d/packages/core/src/components/popover/popover.tsx#L599 - but I could not see any difference setting a padding in the example I tried.  We can try in the app where we see this issue.